src/main/java/com/example/SuperHero.java

```diff
@@ -0,0 +1,18 @@
+package com.example;
+
+public class SuperHero {
+
+    private final String name;
+
+    public SuperHero(String name) {
+        this.name = name;
+    }
+
+    public void saveTheWorld() {
+        System.out.println(this.name + " is saving the world.");
+    }
+
+    public void fly() {
```

Comment:
Not all superheroes can fly; this shouldn't belong in this class - we should have e.g. a `CanFly` interface that flying superheroes can implement?